rules/windows/privilege_escalation_unusual_printspooler_childprocess.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/07/06"
+maturity = "production"
+updated_date = "2021/07/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects unusual Print Spooler service spoolsv.exe child processes. This may indicate an attempt to exploit privilege
+escalation vulnerabilities related to the Printing Service on Windows.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Unusual Print Spooler Child Process"
+references = [
+    "https://msrc.microsoft.com/update-guide/vulnerability/CVE-2021-34527",
+    "https://github.com/afwu/PrintNightmare"
+]
+risk_score = 47
+rule_id = "ee5300a7-7e31-4a72-a258-250abb8b3aa1"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.action == "start" and
+ process.parent.name : "spoolsv.exe" and user.id : "S-1-5-18" and
+ not process.name : ("splwow64.exe", "PDFCreator.exe", "acrodist.exe", "spoolsv.exe", "msiexec.exe", "route.exe", "WerFault.exe") and
```

Comment:
```suggestion

 /* exclusions for FP control below */
 not process.name : ("splwow64.exe", "PDFCreator.exe", "acrodist.exe", "spoolsv.exe", "msiexec.exe", "route.exe", "WerFault.exe") and
```

Reply:
should probably add some verbiage on `false_positives` for this one (and any others that need it).

Reply:
added FP note.